packages/orchestrator-ui-components/tsconfig.json

```diff
@@ -1,9 +1,11 @@
 {
     "extends": "@orchestrator-ui/tsconfig/base.json",
     "compilerOptions": {
-        "strictNullChecks": true
+        "strictNullChecks": true,
+        "rootDir": "./src",
```

Comment:
Do you think we should also apply this to the other packages?

Reply:
It should only apply to packages containing typescript. Even the package @orchestrator-ui/tsconfig is not needed, as it only exports json files.